prov/net/src/xnet_progress.c

```diff
@@ -1345,6 +1346,8 @@ static int xnet_init_uring(struct xnet_uring *uring, size_t entries,
 		return ret;
 
 	uring->fid.fclass = XNET_CLASS_URING;
+	uring->sockapi = sockapi;
+	uring->sockapi->credits = ofi_uring_sq_space_left(&uring->ring);
```

Comment:
This change is in the wrong patch, or should be in a new patch, or needs some description in the commit message regarding why it's needed.

Reply:
Thanks! I isolated this change in a new patch.